### PR TITLE
Versions bumped

### PR DESCRIPTION
**Pull request by iggbom**
Curity SDK -> 7.0.0
Java -> 17
Kotlin -> 1.6.10
org.hibernate.validator:hibernate-validator -> 7.0.3.Final
Replaced deprecated NotBlank annotation with javax.validation.constraints.NotBlank

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -84,9 +84,9 @@
             <scope>provided</scope>
         </dependency>
         <dependency>
-            <groupId>org.hibernate</groupId>
+            <groupId>org.hibernate.validator</groupId>
             <artifactId>hibernate-validator</artifactId>
-            <version>6.2.0.CR1</version>
+            <version>7.0.3.Final</version>
```
Comment (daniellindau) [inline pom.xml:99]: This is not the version in the server is it? Could cause problems in runtime

**Review by daniellindau — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (mtrojanowski) [reply pom.xml:99]: Current docs show that this is the provided version.

File: pom.xml [hunk 2]
```diff
@@ -13,9 +13,9 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <project.curityVersion>6.7.0</project.curityVersion>
+        <project.curityVersion>7.0.0</project.curityVersion>
         <project.slf4jVersion>1.7.30</project.slf4jVersion>
```
Comment (mtrojanowski) [inline pom.xml]: The current provided version is 1.7.36

**Review by mtrojanowski — APPROVED** [review]

File: pom.xml (hunk 2 above)
Comment (iggbom) [reply pom.xml]: updated to 1.7.36

**Review by mtrojanowski — APPROVED** [review]

**Pull request merged**